components/brave_rewards/resources/extension/brave_rewards/components/panel.tsx

```diff
@@ -331,6 +331,46 @@ export class Panel extends React.Component<Props, State> {
     }
   }
 
+  onContributionAmountChange = (publisherKey: string, newAmount: string) => {
+    const newValue = parseInt(newAmount, 10)
+
+    if (newValue === 0) {
+      this.actions.removeRecurringContribution(publisherKey)
+    } else {
+      this.actions.saveRecurringDonation(publisherKey, newValue)
+    }
+  }
+
+  generateAmounts = () => {
+    const { rates } = this.props.rewardsPanelData.walletProperties
+
+    return [0, 1, 5, 10].map((value: number) => {
```

Comment:
we need to get amounts from publisher list and if it's empty default back

Reply:
@NejcZdovc these are amounts are now retrieved from publisher donation info, if no custom values were set, it defaults to this list.